rclcpp/include/rclcpp/node_impl.hpp

```diff
@@ -20,6 +20,9 @@
 #include <string>
 
 #include <rmw/rmw.h>
+#include <rmw/get_type_support_handle.h>
+#include <rmw/get_service_type_support_handle.h>
```

Comment:
They are used in lines 69 (https://github.com/ros2/rclcpp/blob/disambiguate-linking/rclcpp/include/rclcpp/node_impl.hpp#L69) and 100 (https://github.com/ros2/rclcpp/blob/disambiguate-linking/rclcpp/include/rclcpp/node_impl.hpp#L100)